testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -116,6 +116,24 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I see "Selected channels/products were scheduled successfully for syncing." text
     And I wait until I see "SUSE Linux Enterprise Server 15 SP3 x86_64" product has been added
 
+@sle15sp4_minion
+  Scenario: Add SUSE Linux Enterprise Server 15 SP4
+    When I follow the left menu "Admin > Setup Wizard > Products"
+    And I wait until I do not see "Loading" text
+    And I enter "SUSE Linux Enterprise Server 15 SP4 x86_64" as the filtered product description
+    And I select "SUSE Linux Enterprise Server 15 SP4 x86_64" as a product
```

Comment:
This has BETA in the name, check for the exact string.

Reply:
thanks for the catch!